src/frontend/src/pages/PPQFormPage.tsx

```diff
@@ -115,34 +119,23 @@ function PPQFormPage() {
       piaType: piaType === 'null' ? null : piaType,
       containsPersonalInformation: containsPI === 'Yes' ? true : false,
       proposedStartDate: startDate,
+      ...checkedPIItems,
     };
-    const requestBody = { ...bodyPart1, ...checkedPIItems };
     try {
-      const res = await fetch(
-        `http://${import.meta.env.VITE_REACT_API_HOST}:${
-          import.meta.env.VITE_REACT_API_PORT
-        }/api/ppq`,
+      const res = await client(
```

Comment:
can you also rename this to somethign like `httpClient`?

Reply:
done